migration-engine/connectors/sql-migration-connector/src/lib.rs

```diff
@@ -237,7 +237,10 @@ impl MigrationConnector for SqlMigrationConnector {
         Box::new(SqlDestructiveChangesChecker { connector: self })
     }
 
-    fn deserialize_database_migration(&self, json: serde_json::Value) -> Result<SqlMigration, DatabaseMigrationFormatChanged> {
+    fn deserialize_database_migration(
+        &self,
+        json: serde_json::Value,
+    ) -> Result<SqlMigration, DatabaseMigrationFormatChanged> {
```

Comment:
Weird, it looks like the line length problem again. We configured a custom length, I don't know if it's taken into account here.

Reply:
The line length here was 137 characters; the limit in `rustfmt.toml` is set to 120 chars. Rustfmt seems to be working as intended. Overall I'd love to get CI setup to catch this since it would allow us to delegate any ambiguity about lints to "whatever CI says we should do".

Reply:
:+1:

I'm in favour of having this in CI as well.

Reply:
cool, seems like this needs to be updated in a separate repo which @dpetrick is picking up. Think this should be good to merge then!